runtime-k3d/src/runtime/controlplane.py

```diff
@@ -120,18 +122,18 @@ def create_cluster(
     else:
         spinner.write("> Creating cluster without proxy configuration.")
 
+    services_to_expose = ["vehicledatabroker", "mqtt-broker", "seatservice"]
```

Comment:
Why not iterating over all services? Basically we should expose all defined ports or?